src/blocks/donate/frontend/class-newspack-blocks-donate-renderer-base.php

```diff
@@ -246,6 +246,20 @@ protected static function render_streamlined_payment_ui( $attributes ) {
 							value="<?php echo esc_attr( $user_display_name ); ?>"
 						>
 					</div>
+					<div class="stripe-payment__row stripe-payment__row--additional-fields">
+						<?php foreach ( $attributes['additionalFields'] as $field ) : ?>
+							<input
+								data-is-additional-field
+								type="<?php echo esc_attr( $field['type'] ); ?>"
+								name="<?php echo esc_attr( $field['name'] ); ?>"
+								placeholder="<?php echo esc_attr( $field['label'] ); ?>"
+								style="width: calc(<?php echo esc_attr( $field['width'] ); ?>% - 0.5rem);"
```

Comment:
This probably only happened to me because I had fields created before you added the `width` property, but I saw a fatal at first for fields that didn't have a set width. The fatal did not happen after I deleted and recreated the fields.

```suggestion
								<?php if ( isset( $field['width'] ) ) : ?>
								style="width: calc(<?php echo esc_attr( $field['width'] ); ?>% - 0.5rem);"
								<?php endif; ?>
```

Reply:
Yep, that's expected. The feature is not merged yet so there's no need to handle "legacy" fields.